src/plugins/intel_gpu/tests/test_cases/convolution_gpu_test.cpp

```diff
@@ -9071,6 +9071,7 @@ TEST_P(convolution_gpu_onednn, conv_onednn_cases) {
         conv_fsv.output_paddings = {padding({ 0, 0, 0, 0 }, 0.f)};
         topology.add(conv_fsv);
     }
+    topology.add(reorder("reorder_bfyx", input_info("conv_fsv"), format::bfyx, data_types::f32));
```

Comment:
why is `reorder_bfyx` needed?

Reply:
This test case is in WIP. As far as I know, output node has some special treatment. Therefore I separated convolution node and output node to avoid such special treatment.

Reply:
Why did you update the case if it is WIP?

Reply:
For example, [this](https://github.com/openvinotoolkit/openvino/pull/14837/files#diff-41a32bc38b6fa9c7db5e656050c380c993d424df879bb6a1e2627e7008a7704bR1866) patch is related to the side-effect of output node. Moreover, many other testcases are also using reorder_bfyx as an output node (possibly for the same purpose as above), and there is no reason that convolution node has to be output node in this testcase. Besides, #14860(new PR of #14143) is not WIP now (in review). Therefore I strongly recommend this change for the view of consistency and safety.